client/pom.xml

```diff
@@ -26,6 +26,31 @@
             <artifactId>resteasy-client-microprofile</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-admin-client</artifactId>
+            <version>10.0.1</version>
+            <exclusions>
+                <exclusion>
```

Comment:
Just curious, what's the reason for these exclusions?

Reply:
Keycloak admin client uses version `3.9.1.Final` for rest-easy client. Which is conflicting with the versions already included by micro profile client (4.X) & other dependencies.